src/cypher/run.py

```diff
@@ -198,11 +198,16 @@ def get_graph(self, directed=True):
             graph = nx.MultiGraph()
         for item in self._results.graph:
             for node in item['nodes']:
-                graph.add_node(node['id'],
-                               **{**node['properties'], **{'labels': node['labels']}})
+                properties = node['properties'].copy()
```

Comment:
Don't want to be picky, but maybe it's better to use a deep copy instead of a shallow copy so we avoid possible side-effects. Something like:
```python
import copy
...
                properties = copy.deepcopy(node['properties'])
```
And the same for `properties = rel['properties']`